libs/perun/services/src/lib/ApiInterceptor.ts

```diff
@@ -133,4 +136,11 @@ export class ApiInterceptor implements HttpInterceptor {
     rpcError.payload = req.body as unknown as object;
     return rpcError;
   }
+
+  private isNotConsolidatorOrLinker(): boolean {
+    return !(
+      this.store.getProperty('application') === 'Linker' ||
+      this.store.getProperty('application') === 'Consolidator'
+    );
```

Comment:
```suggestion
    const application = this.store.getProperty('application');
    return !(application  === 'Linker' || application ) === 'Consolidator');
```

Reply:
In my opinion the condition `!(application  === 'Linker' || application ) === 'Consolidator')` seems super not human readable, so I would like to stay it the way it is